src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,318 @@
+.. _Poincare operator:
+
+Poincare operator
+~~~~~~~~~~~~~~~~~
+
+The Poincare operator constructs a Poincaré section for toroidal geometry, i.e.
+for streamlines/fieldlines that lie on toroidal geometry (cylindrical
+coordinates). The basis of constructing a connected plot is to accurately
+determine the number of toroidal and poloidal windings (i.e. the winding pair).
+This process is iterative, starting with a minimum number of puncture points
+and continuing until the toroidal and poloidal windings are known or the
+maximum number of punctures is reached. If an accurate winding pair is
+determined, then the puncture points are connected based on it. For more
+information, refer to the following resources.
+
+A.R. Sanderson, G. Chen, X. Tricoche, E. Cohen. “Understanding Quasi-Periodic
```

Comment:
Are these refs linkable on line? If not, might @ARSanderson have PDFs he can give us and we can include like we do with the *Verdict Manual* in [this section](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/gui_manual/Quantitative/Expressions.html#mesh-quality-expressions)